app/components/govuk_component/summary_list_component/action_component.rb

```diff
@@ -1,7 +1,7 @@
 class GovukComponent::SummaryListComponent::ActionComponent < GovukComponent::Base
-  attr_reader :href, :text, :visually_hidden_text, :attributes, :classes
+  attr_reader :href, :text, :visually_hidden_text, :action_suffix, :attributes, :classes
```

Comment:
Think I'd call this `card_title` too, to better match [the nunjucks implementation](https://github.com/alphagov/govuk-frontend/blob/main/packages/govuk-frontend/src/govuk/components/summary-list/template.njk#L7)?

```suggestion
  attr_reader :href, :text, :visually_hidden_text, :card_title, :attributes, :classes
```